web/src/i18n/locales/ja.json

```
@@ -3133,6 +3133,135 @@
     "默认折叠侧边栏": "サイドバーをデフォルトで折りたたむ",
     "默认测试模型": "デフォルトテストモデル",
     "默认用户消息": "こんにちは",
-    "默认补全倍率": "デフォルト補完倍率"
+"默认补全倍率": "デフォルト補完倍率",
```

Biome (2.4.4)

[error] The key 默认补全倍率 was already declared.

(lint/suspicious/noDuplicateObjectKeys)